db/backup_create.go

```diff
@@ -0,0 +1,21 @@
+package db
+
+import (
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	errgo "gopkg.in/errgo.v1"
+)
+
+func CreateBackup(app, addon string) error {
+	client, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+	err = client.BackupCreate(app, addon)
+	if err != nil {
+		return err
+	}
+
+	io.Status("Successfully ordered to make a backup")
```

Comment:
It would be nice if we could follow the backup progression.

Reply:
Well actually it was not really well designed. The backend should have returned `202` and an operation URL in the `Location` header. Then we could have follow the process... ^^

That being said I can add a `backups-show` command and update the status message to state:

`Successfully scheduled a new backup. Type "scalingo backups-show backup-id" to follow the progress`

If it's what you want, I would prefer to do it in a separate PR. OK?

Reply:
```
	for !backup.IsFinished() {
		spinner.Lock()
		if backup.Status == types.BackupStatusScheduled {
			spinner.Suffix = " Waiting for the backup to start"
		} else {
			spinner.Suffix = " Waiting for the backup to finish"
		}
		spinner.Unlock()

		time.Sleep(1 * time.Second)

		backup, err = client.BackupGet(dbId, backup.ID.Hex())
		if err != nil {
			return errors.Wrap(err, "Fail to refresh backup state")
		}
	}
```

This is what i did for the db-cli. I think that this might be enough no ?

Reply:
Do you mean `BackupGet` or `GetBackup`? :laughing:

Reply:
But yes, good idea!